.github/workflows/trunk.yml

```diff
@@ -175,6 +175,26 @@ jobs:
           { config: "force_on_cpu", shard: 1, num_shards: 1, runner: "windows.4xlarge.nonephemeral" },
         ]}
 
+  linux-jammy-py3-clang12-executorch-build:
+    name: linux-jammy-py3-clang12-executorch
```

Comment:
Why did the build step take 1hr 20 mins to run? 
https://github.com/pytorch/pytorch/actions/runs/6868648418/job/18679679827?pr=113364

Reply:
Let me take a look.  There are Docker build time which is an one time cost there, but the build does take a bit longer than usual (~55m)

Reply:
> Let me take a look. There are Docker build time which is an one time cost there, but the build does take a bit longer than usual (~55m)

How long is considered normal?

Reply:
With caching, it usually takes half an hour or less for CPU build.  I could simply retry the job first to see if the build time drops

Reply:
Caching is kicking in now, and the build job takes only 16m now https://github.com/pytorch/pytorch/actions/runs/6872890165/job/18692077299